src/Adapter/Import/RequestPreparator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Import;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class RequestPreparator prepares the request for legacy import controller.
+ */
+class RequestPreparator
```

Comment:
i think that this (or very similar) behavior will be needed even for migrated Import, right? so i suggest adding interface for it and making it final.

also, as it does not depend on any adapter services, you can consider moving it to `Core`?

Reply:
actually this class is no longer needed.

Reply:
even better that way. :D